notebooks/compare_esm_ssp_opt.ipynb

```diff
@@ -0,0 +1,691 @@
+{
```

Comment:
<p>I think we should be able to use <a href="https://esmlab.readthedocs.io/en/latest/api.html#esmlab.core.resample" target="_blank">https://esmlab.readthedocs.io/en/latest/api.html#esmlab.core.resample</a> for this now.</p><p> </p><p>cc @andersy005</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/ctsm_py/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>I tried to apply <code>esmlab.core.resample</code> as follows, but I ran into issues, perhaps because resample expects a dataset instead of a dataarray?</p><p> </p><p><code>test_nbp_amean = esmlab.core.resample(nbpds['C4MIP.CCCma.CanESM5.esm-ssp585.Lmon.gn']['nbp'],freq='mon',weights=mon_wgts) * SEC_PER_YEAR</code></p><p> </p><pre class="ql-syntax" spellcheck="false">Error: AttributeError: 'DataArray' object has no attribute 'esmlab'
</pre><p> </p>

Reply:
<p>I also just noticed that the resample documentation has something about the weights argument being supported for annual means only.</p>